src/parser/core/modules/BuffWindow.tsx

```diff
@@ -302,6 +302,25 @@ export abstract class BuffWindowModule extends Module {
 		return undefined
 	}
 
+	private countMissedTrackedActions(buffWindow: BuffWindowState, action: BuffWindowTrackedAction): number {
+		const expected = this.getBuffWindowExpectedTrackedActions(buffWindow, action)
+		const actual = buffWindow.getActionCountByIds([action.action.id])
+		const comparator = this.changeComparisonClassLogic(buffWindow, action)
```

Comment:
This is a little hard to follow what's going on here.  I'm not sure I have a better alternative at hand that could still share / reuse the rotation table's highlighting logic, but it took me 15 solid minutes to grok this.  Might be worth some further investigation / discussion of whether the RotationTableOutcome interface can be simplified somehow in the future?